### DISCUSSION TITLE
captionfieldID for "ND-ExclusionGrounds"

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#881] [Ideas]
Is it possible to add a captionfieldID for "ND-ExclusionGrounds"? In my opinion, "BT-67(a)-Procedure" would be a suitable ID. This would greatly improve the display of notices with many exclusion grounds and thus significantly simplify the recording.

**Comment by YvesJo** [comment]
Hi,
CaptionfieldIDs are useful when a reference is made to another field like it is done for the Lot and a more user friendly characteristic text is displayed instead of a technical ID. With BT-67, based on a codelist we do not face such a situation.

**Reply by JanEmde** [reply]
Ah, ok. We also use the ID to make repeatable elements easier for the user to identify:
![grafik](https://github.com/OP-TED/eForms-SDK/assets/144460224/a6019d4e-a7c5-4783-8848-fbc6466f846d)

By adding the ID, we could do the same for the exclusion grounds. Currently, the repeatable elements (without CaptionFieldID) always have to be expanded first in order to recognise what exactly the content is:
![grafik](https://github.com/OP-TED/eForms-SDK/assets/144460224/7ab826f4-5b01-4851-8db8-71ccbcd5a4b6)

Of course, we could also build a special solution in our frontend for this case, but a joint solution directly in the SDK would make a lot of sense in my view.
